src/pages/admin/history/components/history-page-content/HistoryPageContent.module.scss

```
@@ -51,4 +51,156 @@
+    background-color: #fff;
+    min-height: 130px;
+    min-width: 700px;
+    padding-top: 30px;
+}
+
+.actions {
+    display: flex;
+    justify-content: center;
+    gap: 32px;
+
+    Button {
```

Stylelint (17.6.0)

[error] Expected "Button" to be "button" (selector-type-case)

(selector-type-case)